apps/web/modules/cache/redis.test.ts

```
@@ -0,0 +1,261 @@
+    process.env.REDIS_URL = "redis://localhost:6379";
+
+    // Reset isReady state
+    mockRedisClient.isReady = true;
+
+    // Make connect resolve successfully
+    mockRedisClient.connect.mockResolvedValue(undefined);
+  });
+
+  afterEach(() => {
+    vi.resetModules();
+    delete process.env.REDIS_URL;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)